ENGAGEHF/Dashboard/Notifications/Views/NotificationSection.swift

```diff
@@ -0,0 +1,80 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import FirebaseCore
+import FirebaseFirestore
+import Foundation
+import Spezi
+import SpeziViews
+import SwiftUI
+
+
+struct NotificationSection: View {
+    @Environment(NotificationManager.self) private var notificationManager
+    
+    
+    var body: some View {
+        @Bindable var notificationManager = notificationManager
```

Comment:
If we don't make the notifications Bindings we could go ahead and remove the Bindable here.